src/db/tables/subscribers.ts

```diff
@@ -429,6 +414,83 @@ async function getSubscribersWaitingForMonthlyEmail(
 }
 /* c8 ignore stop */
 
+// Not covered by tests; mostly side-effects. See test-coverage.md#mock-heavy
+/* c8 ignore start */
+async function getFreeSubscribersWaitingForMonthlyEmail(): Promise<
+  SubscriberRow[]
+> {
+  const flag = await getFeatureFlagData("MonthlyReportFreeUser");
+  const accountCutOffDate = parseIso8601Datetime(
+    process.env.MONTHLY_ACTIVITY_FREE_EMAIL_ACCOUNT_CUTOFF_DATE ??
+      "2022-01-01T00:00:00.000Z",
```

Comment:
Ticket said:

> Only send to monitor users (...) who have signed up for monitor in the past 3 years.

However, I'm assuming that that should be 4 years in a year, so I just set 1 January 2022 as the cutoff date.

Reply:
Should this be in `constants.ts`?

Reply:
Constants in `constants.ts` are not overridable at runtime with an environment variable; I figured this would give us some more flexibility if, after deployment, it suddenly turns out that this cutoff date isn't actually the right one.